packages/wallet/src/redux/state.ts

```diff
@@ -77,7 +78,15 @@ export function waitForAdjudicator(params: Properties<WaitForAdjudicator>): Wait
 }
 
 export function initialized(params: Properties<Initialized>): Initialized {
-  const { outboxState, fundingState, channelState, uid, networkId, adjudicator } = params;
+  const {
+    outboxState,
+    fundingState,
+    channelState,
+    uid,
+    networkId,
+    adjudicator,
+    consensusLibrary,
```

Comment:
I think the `consensusLibrary` should be an environment variable.

Reply:
`consensusLibrary` is dependent on the current `networkID` so I don't think it makes sense to pull that out  to an environment variable until we pull out `networkId` and `adjudicator` as well and enforce a single `networkId`